generate_test_reports.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+tests=(NetCore31 Net50 Net60 NetFramework35)
+
+for test in "${tests[@]}"; do
+    test_folder="EasyPost.Tests.$test"
+    cd "$test_folder"
+    dotnet test --collect:"XPlat Code Coverage"
+    reportgenerator -reports:TestResults/*/coverage.cobertura.xml -targetdir:../coveragereport/$test -reporttypes:Html
```

Comment:
This will fail if `dotnet` or `reportgenerator` are not installed. We should add a check prior to this running that checks if they are installed and error if not. We should also call it out in the new section you're adding to the README.

Please also run `shellcheck` on this file (you'll need to download it), there are various items that need correcting to be compliant with POSIX sh.

Reply:
Resolved in [75bac9b](https://github.com/EasyPost/easypost-csharp/pull/222/commits/75bac9bec61a02b4011d5e788bbb8a51b84f059d)